recipes/rmdir/src/workflow.ts

```diff
@@ -1,3 +1,9 @@
+// jssg din't work correclty in npm workspaces, so we use relative imports
```

Comment:
I believe these won't work when the package is published

Reply:
I know I had worked on this issue on upstream and discussed a lot with mohab to fix that